tools/number-converter/firebase.json

```diff
@@ -0,0 +1,17 @@
+{
+	"hosting": {
+		"site": "json-formatter-1ae13",
```

Comment:
We should follow the guide we've made here to ensure we're following a safe process, otherwise it's easy to accidentally overwrite our sites if we deploy:

https://github.com/holistic-web/holistic-toolbox/blob/master/.project/processes/creating-new-tool.md

Let's make sure this doc is up to date and reflects how we make new tools.